store/src/app/shared/components/page-not-found/page-not-found.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-page-not-found',
+  templateUrl: './page-not-found.component.html',
+  styleUrls: ['./page-not-found.component.scss']
+})
+export class PageNotFoundComponent {
+
+  buttonName: string = 'Back to home';
```

Comment:
You can pass string parameters right in the HTML, if it is not dynamic (like in your case)

Reply:
done